library/objective-c/EnvoyEngine.h

```diff
@@ -143,11 +142,10 @@ typedef NSDictionary<NSString *, NSArray<NSString *> *> EnvoyHeaders;
 /**
  Create a new instance of the configuration.
  */
-- (instancetype)initWithDomain:(NSString *)domain
-                   statsDomain:(NSString *)statsDomain
-         connectTimeoutSeconds:(UInt32)connectTimeoutSeconds
-             dnsRefreshSeconds:(UInt32)dnsRefreshSeconds
-             statsFlushSeconds:(UInt32)statsFlushSeconds;
+- (instancetype)initWithStatsDomain:(NSString *)statsDomain
```

Comment:
I think we have a templating bug. Which is that now we are not setting SNI in the template if we don't populate domain. Which actually brings the larger question of how we want to do that (and relatedly #630) for the dynamic proxy? I need to read the docs to see if that can be dynamic as well.

Reply:
According to the docs here it sounds like SNI will happen automatically as long as there's a TLS context on the cluster:
https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/http/http_proxy

Reply:
Will update.